source/Armament.cpp

```diff
@@ -86,7 +97,7 @@ int Armament::Add(const Outfit *outfit, int count)
 			else
 				++existing;
 		}
-		else if(!hardpoint.GetOutfit() && hardpoint.IsTurret() == isTurret)
+		else if(!hardpoint.GetOutfit() && (hardpoint.IsTurret() == isTurret || hardpoint.IsPylon() == isPylon))
```

Comment:
It was correct before.
```suggestion
		else if(!hardpoint.GetOutfit() && hardpoint.IsTurret() == isTurret && hardpoint.IsPylon() == isPylon)
```